### PR TITLE
[Sqlitecpp] update to 3.2.0

### PR DESCRIPTION
**Pull request by SvenPStarFinanz**
**Describe the pull request**

- #### What does your PR fix?
  Updating Sqlitecpp to the current version.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  No change.

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  Yes.

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  Yes.

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/sqlitecpp/portfile.cmake [hunk 1]
```diff
@@ -37,4 +38,5 @@ vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/SQLiteCpp)
 vcpkg_copy_pdbs()
 
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
```
Comment (JonLiu1993) [inline ports/sqlitecpp/portfile.cmake:40]:
```suggestion
file(REMOVE_RECURSE “${CURRENT_PACKAGES_DIR}/debug/include")
```

File: ports/sqlitecpp/portfile.cmake [hunk 2]
```diff
@@ -37,4 +38,5 @@ vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/SQLiteCpp)
...
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
 file(INSTALL ${SOURCE_PATH}/LICENSE.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
```
Comment (JonLiu1993) [inline ports/sqlitecpp/portfile.cmake:42]:
```suggestion
vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE.txt")
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by JonLiu1993** [comment]
@SvenPStarFinanz , Thanks for your pr, could you please consider migrating them to the new functions:
````
vcpkg_install_cmake -> vcpkg_cmake_install (from port vcpkg-cmake)
vcpkg_configure_cmake -> vcpkg_cmake_configure (Please remove the option PREFER_NINJA) (from port vcpkg-cmake)
vcpkg_fixup_cmake_targets -> vcpkg_cmake_config_fixup (from port vcpkg-cmake-config)
````

File: ports/sqlitecpp/portfile.cmake [hunk 3]
```diff
@@ -29,12 +29,14 @@ vcpkg_configure_cmake(
...
         ${USE_STACK_PROTECTION}
 )
 
 vcpkg_install_cmake()
 vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/SQLiteCpp)
 vcpkg_copy_pdbs()
 
-file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
-file(INSTALL ${SOURCE_PATH}/LICENSE.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
+file(REMOVE_RECURSE “${CURRENT_PACKAGES_DIR}/debug/include")
```
Comment (JonLiu1993) [inline ports/sqlitecpp/portfile.cmake:40]:
```suggestion
file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
```

Comment (JonLiu1993) [reply ports/sqlitecpp/portfile.cmake:40]: Note the form of double quotes.

File: ports/sqlitecpp/portfile.cmake [hunk 4]
```diff
@@ -29,12 +29,14 @@ vcpkg_configure_cmake(
         -DSQLITECPP_INTERNAL_SQLITE=OFF
         -DSQLITE_ENABLE_COLUMN_METADATA=OFF
         -DSQLITECPP_INTERNAL_SQLITE=OFF
+		-DSQLITECPP_USE_STATIC_RUNTIME=OFF
```
Comment (JonLiu1993) [inline ports/sqlitecpp/portfile.cmake:32]:
```suggestion
        -DSQLITECPP_INTERNAL_SQLITE=OFF
        -DSQLITECPP_USE_STATIC_RUNTIME=OFF
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/sqlitecpp/vcpkg.json [hunk 5]
```diff
@@ -1,9 +1,21 @@
...
-  "port-version": 2,
+  "version": "3.2.0",
   "description": "SQLiteC++ (SQLiteCpp) is a smart and easy to use C++ SQLite3 wrapper.",
   "homepage": "https://github.com/SRombauts/SQLiteCpp",
+  "license": "MIT",
+  "dependencies": [
+    {
+      "name": "vcpkg-cmake",
+      "host": true,
+      "platform": "windows"
```
Comment (JonLiu1993) [inline ports/sqlitecpp/vcpkg.json:11]:
```suggestion
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Review by JonLiu1993 — DISMISSED** [review]

**Comment by JonLiu1993** [comment]
All features are tested successfully in the following triplet:
- x86-windows
- x64-windows

**Review by BillyONeal — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thank you!

**Pull request merged**